superset-frontend/src/common/components/ProgressBar.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { styled } from '@superset-ui/core';
+import { Progress as AntdProgress } from 'src/common/components';
```

Comment:
I'd like to nudge superset devs toward using only this styled ProgressBar and not the one from Antd directly - just to keep things consistent. What do you think about importing from `antd` directly here, and NOT export `Progress` from common/components/index.tsx?

Reply:
... you could even import this component from `common/components/index.tsx` and export it as `Progress` - then people could `import { Progress } from 'src/common/components'` as it if were the AntD one, but it'd really be this enhanced/styled one.

Reply:
I would rather go with your first suggestion and import the Antd Progress directly into the enhanced component. From src/common/components only export the enhanced Badge so that accessing the Antd one directly becomes unlikely/harder.